test/common/upstream/eds_test.cc

```diff
@@ -228,6 +228,17 @@ TEST_F(EdsTest, ValidateFail) {
   EXPECT_FALSE(initialized_);
 }
 
+// Validate that onConfigUpdate() can ignore unknown fields.
+TEST_F(EdsTest, ValidateIgnored) {
+  validation_visitor_.setSkipValidation(true);
+  initialize();
+  envoy::config::endpoint::v3::ClusterLoadAssignment resource;
+  Protobuf::RepeatedPtrField<ProtobufWkt::Any> resources;
+  resources.Add()->PackFrom(resource);
+  // initialization fails but we don't throw, unlike ValidateFail above
+  EXPECT_FALSE(initialized_);
```

Comment:
Should we verify that no stats were bumped or something like that?

Reply:
I've modified the integration test to validate that the counters aren't incremented.